app/src/main/java/com/example/bigowlapp/activity/SendSmsInvitationActivity.java

```diff
@@ -18,18 +19,38 @@
 public class SendSmsInvitationActivity extends AppCompatActivity {
 
     private EditText number, message;
-    private Button send, testSendRequest;
-    private String number2, message2;
+    private Button send;
+    private String number2, message2, noteText;
+    private TextView noteTv;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_send_sms_invitation);
 
-        number = findViewById(R.id.number);
+        noteTv = (TextView) findViewById(R.id.note);
+
+        String contactDetails = getIntent().getStringExtra("number1");
```

Comment:
By fixing the above, the number1 should become more descriptive.

Reply:
I has been fixed